core/src/createTextMaskInputElement.js

```diff
@@ -196,12 +196,14 @@ export default function createTextMaskInputElement({
           maskRejection: someCharsRejected
         })
       }
+
+      return inputElement // This is not necessary, but it's another option
     }
   }
 }
 
 function safeSetSelection(element, selectionPosition) {
-  if (document.activeElement === element) {
+  if (!document || document.activeElement === element) {
```

Comment:
You mean there could be a global variable called `document` and it may not have `activeElement`?